cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
The `validate` method in `TicketSerializer` calls a class method on the `Ticket` model to perform validation. While this works, it would be more idiomatic in Django REST Framework to perform the validation in the serializer itself. This would make the validation errors more clear and easier to handle.